spec/models/order_balance_spec.rb

```diff
@@ -325,6 +325,40 @@
     end
   end
 
+  describe '#to_f and #to_d' do
```

Comment:
Would youind duplicating them? I think is best to make them independent so both we don't tie them innecessarely. We could use an RSpec's shared examole if needed.

Reply:
I started off with them separated, but I merged them as the huge amount of duplication seemed like it wasn't worth it for these quick tests :see_no_evil:

Reply:
shared examples could help removing the duplication.